db/schema.rb

```diff
@@ -433,15 +433,15 @@
 
   add_index "roles", ["slug"], name: "index_roles_on_slug", unique: true, using: :btree
 
-  create_table "scientific_names", force: true do |t|
-    t.string   "scientific_name", null: false
-    t.integer  "crop_id",         null: false
+  create_table "scientific_names", force: :cascade do |t|
+    t.string   "name",       null: false
+    t.integer  "crop_id",    null: false
     t.datetime "created_at"
     t.datetime "updated_at"
     t.integer  "creator_id"
   end
```

Comment:
Its interesting the lack of indexes on crop_id, etc in the schema. Should fix that in another ticket probably